extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -148,6 +160,45 @@ export class IframeTransport {
   }
 
   /**
+   * Uses the Long Task API to create an observer for when 3p vendor frames
+   * take more than 50ms of continuous CPU time.
+   * Currently the only action in response to that is to log. There is a
+   * grace of LONG_TASK_REPORTING_THRESHOLD_ occurrences before logging begins.
+   * @VisibleForTesting
+   */
+  createLongTaskObserver() {
```

Comment:
I don't believe this has to be public function even with the related test. Since we run unit test against non compiled code, you can also spy `createLongTaskObserver_`

Reply:
Done